pkg/kubernetes/node/v1alpha1/node_test.go

```diff
@@ -0,0 +1,179 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func fakeEmptyNodeList(noOfNodes int) *corev1.NodeList {
```

Comment:
according to idiomatic maya guideline this should be `fakeNodeListEmpty`

Reply:
Updated func name